src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -509,21 +531,33 @@
   [message-content-wrapper message
    [unknown-content-type message]])
 
-(defn chat-message [message space-keeper]
-  [reactions/with-reaction-picker
-   {:message         message
-    :reactions       @(re-frame/subscribe [:chats/message-reactions (:message-id message)])
-    :picker-on-open  (fn []
-                       (space-keeper true))
-    :picker-on-close (fn []
-                       (space-keeper false))
-    :send-emoji      (fn [{:keys [emoji-id]}]
-                       (re-frame/dispatch [::models.reactions/send-emoji-reaction
-                                           {:message-id (:message-id message)
-                                            :emoji-id   emoji-id}]))
-    :retract-emoji   (fn [{:keys [emoji-id emoji-reaction-id]}]
-                       (re-frame/dispatch [::models.reactions/send-emoji-reaction-retraction
-                                           {:message-id        (:message-id message)
-                                            :emoji-id          emoji-id
-                                            :emoji-reaction-id emoji-reaction-id}]))
-    :render          ->message}])
+(defview chat-message [message space-keeper]
+  (letsubs [scale-anim-value (animation/create-value 0.25)
+            translate-x-anim-value (animation/create-value 120)
```

Comment:
hey @briansztamfater awesome job on the animation! can you tell me where those values for x/y translation come from?

Reply:
Hi @errorists! As we cannot natively set an anchor point / transform-origin for making the scaling animation start from the bottom-right of the view (it will always scale from the mid-x / mid-y), I set some arbitrary translate values to give that impression of movement while scaling. I tried some values and liked the results, but they can be adjusted.

I found some articles with workarounds for this but didn't want to over-engineer if the results were good enough: https://suelan.github.io/2020/05/15/20200515-react-native-anchor-point/

Reply:
cool. I'd make sure that this solution scales across screen sizes as message width is variable. so I'm a bit concerned that a one-size fits all magic number might not cut it and some relative % computation might be better. having tested it on an iPhone 12 viewport it looked absolutely fine tho so maybe I'm overthinking it

Reply:
Sure, I’ll make those values proportional to the screen size and do some further testing with different kind of simulators :)

Reply:
Done -- Tested on iPhone 8, iPhone 11, iPhone 11 Pro and iPhone 11 Pro Max simulators.